diff --git a/app/Http/Controllers/Web/InstallController.php b/app/Http/Controllers/Web/InstallController.php
index 0515da7..d773736 100644
--- a/app/Http/Controllers/Web/InstallController.php
+++ b/app/Http/Controllers/Web/InstallController.php
@@ -5,6 +5,7 @@ namespace Vanguard\Http\Controllers\Web;
 use Artisan;
 use DB;
 use Dotenv\Dotenv;
+use Illuminate\Foundation\Bootstrap\LoadEnvironmentVariables;
 use Illuminate\Http\Request;
 
 use Input;
@@ -119,7 +120,7 @@ class InstallController extends Controller
 
     private function reloadEnv()
     {
-        (new Dotenv(base_path()))->load();
+        (new LoadEnvironmentVariables)->bootstrap(app());
     }
 
     public function complete()
diff --git a/config/app.php b/config/app.php
index f039ecc..e8dac24 100644
--- a/config/app.php
+++ b/config/app.php
@@ -23,7 +23,7 @@ return [
     | Vanguard Version
     |--------------------------------------------------------------------------
     */
-    'version' => '3.2.0',
+    'version' => '3.2.1',
 
     /*
     |--------------------------------------------------------------------------
